packages/fxa-settings/src/pages/ResetPasswordWithRecoveryKeyVerified/index.stories.tsx

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import ResetPasswordWithRecoveryKeyVerified from '.';
+import AppLayout from '../../components/AppLayout';
+import { LocationProvider } from '@reach/router';
+import { Meta } from '@storybook/react';
+
+export default {
+  title: 'pages/ResetPasswordWithRecoveryKeyVerified',
+  component: ResetPasswordWithRecoveryKeyVerified,
+} as Meta;
+
+export const Default = () => (
+  <LocationProvider>
+    <AppLayout>
+      <ResetPasswordWithRecoveryKeyVerified />
```

Comment:
Can `ResetPasswordWithRecoveryKeyVerified` display a different service than settings? If it can, it would be nice to have that as another state here in Storybook, but I think either way, we need states documented for the `Ready` component that I'm not seeing. Fine to do here or in another PR, or even a follow up ticket, since there's probably going to be a lot of different states we need to document.

Reply:
It can, but that's tested in the Ready component.

Reply:
@millsoper Can we add those Ready storybook states in a follow up?